.gitlab/image_deploy/docker_windows.yml

```diff
@@ -37,6 +37,7 @@ dev_branch_docker_hub-a7-windows:
   variables:
     VARIANT: 2004
   script:
+    - '$_instance_id = (iwr  -UseBasicParsing http://169.254.169.254/latest/meta-data/instance-id).content ; Write-Host "Running on instance $($_instance_id)"'
```

Comment:
Is this needed, given that the instance id is already printed in `.docker_tag_windows_job_definition` (which this job is an extension of)?